### DISCUSSION TITLE
Layout Preset/Default Indications

### DISCUSSION THREAD

**Discussion by MentalGear** [directus/directus#17207] [Feature Requests]
I had some trouble figuring out how layout/presets are handled in Directus, and it was quite annoying to have my layout messed up after each logout. I had to study the docs for something I think should be rather idiomatic.

For example: if a user modifies a layout, it should be automatically saved.

Furthermore, the sort of "official hack" to having first to create a preset, then delete the preset's name to make it a default is far from intuitive. It'd be better to have just a clearly labelled checkbox "use as default" on a preset.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
